pyveg/src/satellite_data_analysis.py

```diff
@@ -221,6 +226,7 @@ def get_time_series(num_time_periods,
                     end_date,
                     mask_cloud=False, ## EXPERIMENTAL - false by default
                     output_dir=".",
+                    output_suffix=".",#end of output filename, including file extension" # DOESNT SEEM TO BE USED
```

Comment:
Yep, I think your comment is right - I'm removing some arguments (including this one) in a different branch that we can merge in later..